presto-main/src/main/java/com/facebook/presto/operator/SortedPositionLinks.java

```diff
@@ -152,13 +153,14 @@ public Factory build()
             return new Factory()
             {
                 @Override
-                public PositionLinks create(Optional<JoinFilterFunction> lessThanFunction)
+                public PositionLinks create(Optional<JoinFilterFunction> lessThanFunction, List<JoinFilterFunction> filterFunctions)
```

Comment:
I do not understand why we keep old `lessThanFunction` and add a list of `filterFunctions`.

Would `lessThanFunction` not be on of the functions in `filterFunctions` list?